pom.xml

```diff
@@ -57,6 +57,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
     <java.version>8</java.version>
+    <scala.binary.version>2.12</scala.binary.version>
```

Comment:
we need a default value to make IDEA happy, othewise IDEA can not load moudles correctly.